components/shutter/variable.tf

```diff
@@ -1 +1,4 @@
-../../environments/variable.tf
\ No newline at end of file
+../../environments/variable.tf
+
+variable "pip_shutter_storage" {}
```

Comment:
this is a symlink to `../../environments/variable.tf` you can't edit this file needs to be the one in `../../environments/variable.tf`